contrib/format-maprdb/src/main/java/org/apache/drill/exec/store/mapr/db/json/CompareFunctionsProcessor.java

```diff
@@ -93,7 +95,9 @@ public static CompareFunctionsProcessor processWithTimeZoneOffset(FunctionCall c
       protected boolean visitTimestampExpr(SchemaPath path, TimeStampExpression valueArg) {
         // converts timestamp value from local time zone to UTC since the record reader
         // reads the timestamp in local timezone if the readTimestampWithZoneOffset flag is enabled
-        long timeStamp = valueArg.getTimeStamp() - DateUtility.TIMEZONE_OFFSET_MILLIS;
+        long timeStamp = Instant.ofEpochMilli(valueArg.getTimeStamp()).atZone(ZoneId.of("UTC"))
```

Comment:
This is a long chain of functions .. could you split this into couple of statements ? Helps both readability and debugging.

Reply:
Thanks, done.